sdk/include/ndk/rtltypes.h

```diff
@@ -1889,6 +1889,10 @@ typedef struct _RTL_UNICODE_STRING_BUFFER
 
 #ifndef NTOS_MODE_USER
 
+#ifndef RT_MESSAGETABLE
+#define RT_MESSAGETABLE 11
```

Comment:
This seems confusing to me given that winuser.h defines it differently (namely as a pointer via `MAKEINTRESOURCE`).

Reply:
hm yeah, but this generates a pointer (to string). While RtlFindMessage uses a ULONG only for the resource type.

Reply:
Right, but that means calls to `RtlFindMessage` will look different in UM vs KM, and you have to be aware of which of the two headers you've included in order to know which code to write. That seems more annoying to me than having to add casts in KM code.